internal/build/gen_mk.sh

```
@@ -102,13 +101,13 @@ EOT
 		cat <<-EOT
 		$files$TAB=$TAB$files_cmd
 		EOT
-	done < "$INDEX" | column -t -s "$TAB" -o " "
+	done < "$INDEX" | column -t -s "$TAB"
 }
 
 gen_make_targets() {
 	local cmd="$1" name="$2" dir="$3" mod="$4" deps="$5"
 	local call= callu= callx=
-	local depsx= cmdx=
+	local depsx=
```

Shellcheck

[warning] In POSIX sh, 'local' is undefined.

(SC3043)